src/utils/Event.ts

```diff
@@ -0,0 +1,34 @@
+/// <reference path='../app.ts' />
+
+module n3Charts.Utils {
+  'use strict';
+
+  export class EventManager {
```

Comment:
Hmmmm should this be in an `EventManager.ts` file instead ? I think it's important to have a good consistency between entities and filenames...